demo/app/app.go

```diff
@@ -326,9 +326,11 @@ func NewConsumerApp(
 		app.appCodec,
 		keys[bbntypes.StoreKey],
 		memKeys[bbntypes.MemStoreKey],
+		app.AccountKeeper,
 		app.BankKeeper,
 		app.StakingKeeper,
 		&app.WasmKeeper, // ensure this is a pointer as we instantiate the keeper a bit later
+		authtypes.FeeCollectorName,
 		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
 	)
```

Comment:
I think we also need to put the Babylon module after `x/mint` module but before `x/distirbution` module in `SetOrderBeginBlockers` at L595

Reply:
please check [bdcc0a3](https://github.com/babylonlabs-io/babylon-sdk/pull/164/commits/bdcc0a30d3b7d21f5efdc976d766def77353e5a3)